Tests/private_build/upload_packs_private.py

```diff
@@ -483,7 +484,7 @@ def main():
     upload_id_set(default_storage_bucket, id_set_path)
 
     # summary of packs status
-    print_packs_summary(packs_list)
+    print_packs_summary(packs_list, storage_bucket_name, private_bucket_name)
```

Comment:
you need to update `print_packs_summary`
add unit tests to `print_packs_summary`

Reply:
What is left to update there?
I left out the previous changes on purpose since they are irrelevant, am I missing something here?